sapi_demo/src/EventSubscriber/EventViewEntitySubscriber.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Drupal\sapi_demo\EventSubscriber;
+
+use Drupal\Component\Plugin\PluginManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\EventDispatcher\Event;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\sapi\Dispatcher;
+use Drupal\Core\Routing\CurrentRouteMatch;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Drupal\sapi\ActionTypeInterface;
+
+use Drupal\Core\Entity;
+
+/**
+ * Class EventViewEntitySubscriber.
+ *
+ * @package Drupal\sapi_demo
+ */
+class EventViewEntitySubscriber implements EventSubscriberInterface {
```

Comment:
`EntityViewEventSubscriber` would make a more consistent class name, because it spotlights that it's has to do with `entity` being `viewed` and makes it clear that it's an `EventSubscriber`.